allennlp/data/token_indexers/pretrained_transformer_mismatched_indexer.py

```diff
@@ -58,19 +58,15 @@ def count_vocab_items(self, token: Token, counter: Dict[str, Dict[str, int]]):
     @overrides
     def tokens_to_indices(self, tokens: List[Token], vocabulary: Vocabulary) -> IndexedTokenList:
         orig_token_mask = [1] * len(tokens)
-        tokens, offsets = self._intra_word_tokenize(tokens)
+        tokens, offsets = self._intra_word_tokenize(tokens)  # type: ignore
+        wordpiece_mask = [1] * len(tokens)
 
         # {"token_ids": ..., "mask": ...}
         output = self._matched_indexer.tokens_to_indices(tokens, vocabulary)
```

Comment:
This is the line where I would have expected a `# type: ignore`, from our conversation above.  You changed the type signature of this method in the called class, instead of just add an ignore here.